annotator-scanner/src/main/java/edu/ucr/cs/riple/scanner/SymbolUtil.java

```diff
@@ -154,4 +158,51 @@ private static Stream<? extends AnnotationMirror> getTypeUseAnnotations(Symbol s
     }
     return rawTypeAttributes;
   }
+
+  /**
+   * Locates the region member for the enclosing region at the given path. A region member, can be a
+   * field, method or {@code null} if path leads to a static block initialization.
+   *
+   * @param path Path leading to region.
+   * @param enclosingClass Enclosing class of the path.
+   * @return Region member.
+   */
+  @Nullable
+  public static Symbol locateRegionMemberForSymbolAtPath(
+      TreePath path, Symbol.ClassSymbol enclosingClass) {
+    // Class is not generated by lombok. Let's find the region member, can be a field, method or
```

Comment:
Why is there a comment about lombok here?  Feels somewhat unrelated to what this method is doing

Reply:
Sorry thanks for noticing this. This was copied from existing method. 37387665f141cec005997acb344854d6dea6685a